content/integrations/fastlane-integration.md

```diff
@@ -0,0 +1,43 @@
+title: Fastlane integration
+description: How to use Fastlane in codemagic.yaml
+weight: 4
+---
+
+**Fastlane** is an open source platform aimed at simplifying Android and iOS deployment. If your development team uses Fastlane, it can be used as part of your CI/CD pipeline to build and deploy your applications.
```

Comment:
it would be a good idea to mention that Codemagic has fastlane preinstalled

Reply:
Added